### PR TITLE
[MNOE-282] Comparison of apps in the Marketplace

### PR DESCRIPTION
**Pull request by clemthenem**
@ouranos & @alexnoox

### CODE REVIEW

File: src/app/views/marketplace/marketplace-compare.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            <tr>
+              <th>{{ 'mno_enterprise.templates.dashboard.marketplace.compare.short_description' | translate }}</th>
+              <td ng-repeat="app in vm.apps | filter:{toCompare: true } | limitTo:4">{{::app.tiny_description}}</td>
+            </tr>
+            <tr ng-if="vm.isReviewingEnabled">
+              <th>{{ 'mno_enterprise.templates.dashboard.marketplace.compare.rating' | translate }}</th>
+              <td ng-repeat="app in vm.apps | filter:{toCompare: true } | limitTo:4"><star-rating readonly="true" ng-model="app.average_rating"></star-rating></td>
+            </tr>
+            <tr>
+              <th>{{ 'mno_enterprise.templates.dashboard.marketplace.compare.agencies_number' | translate }}</th>
```
Comment (ouranos) [inline src/app/views/marketplace/marketplace-compare.html]: ~~agencies~~ => companies

File: src/locales/en.locale.json [hunk 2]
```diff
@@ -459,6 +464,15 @@
   "mno_enterprise.templates.dashboard.marketplace.app_selection.select_orga": "Select an organization",
   "mno_enterprise.templates.dashboard.marketplace.app_selection.no_orga_authorized": "Invalid purchase",
   "mno_enterprise.templates.dashboard.marketplace.app_selection.unauthorized_to_add_app": "Unfortunately you do not have permission to purchase products for this organization. Please select another organization.",
+  "mno_enterprise.templates.dashboard.marketplace.compare.title": "Comparison",
+  "mno_enterprise.templates.dashboard.marketplace.compare.application_provider": "Application Provider",
+  "mno_enterprise.templates.dashboard.marketplace.compare.category": "Category",
+  "mno_enterprise.templates.dashboard.marketplace.compare.key_benefits": "Key benefits",
+  "mno_enterprise.templates.dashboard.marketplace.compare.short_description": "Short description",
+  "mno_enterprise.templates.dashboard.marketplace.compare.rating": "Rating",
+  "mno_enterprise.templates.dashboard.marketplace.compare.agencies_number": "Number of agencies using the application",
```
Comment (ouranos) [inline src/locales/en.locale.json]: ~~agencies~~ => companies

**Comment by clemthenem** [comment]
:+1:

File: src/locales/en.locale.json [hunk 3]
```diff
@@ -459,6 +464,15 @@
   "mno_enterprise.templates.dashboard.marketplace.app_selection.select_orga": "Select an organization",
   "mno_enterprise.templates.dashboard.marketplace.app_selection.no_orga_authorized": "Invalid purchase",
   "mno_enterprise.templates.dashboard.marketplace.app_selection.unauthorized_to_add_app": "Unfortunately you do not have permission to purchase products for this organization. Please select another organization.",
+  "mno_enterprise.templates.dashboard.marketplace.compare.title": "Comparison",
+  "mno_enterprise.templates.dashboard.marketplace.compare.application_provider": "Application Provider",
+  "mno_enterprise.templates.dashboard.marketplace.compare.category": "Category",
+  "mno_enterprise.templates.dashboard.marketplace.compare.key_benefits": "Key benefits",
+  "mno_enterprise.templates.dashboard.marketplace.compare.short_description": "Short description",
+  "mno_enterprise.templates.dashboard.marketplace.compare.rating": "Rating",
+  "mno_enterprise.templates.dashboard.marketplace.compare.companies_number": "Number of agencies using the application",
```
Comment (ouranos) [inline src/locales/en.locale.json]: ~~agencies~~ => companies

**Pull request merged**